packages/core/src/components/Tabs/Tabs.jsx

```diff
@@ -4,6 +4,14 @@ import Tab from './Tab';
 import TabPanel from './TabPanel';
 import classnames from 'classnames';
 
+/** CONSTANTS
+ * Adding in the constant values for keycodes
+ * to handle onKeyDown events
+ */
+const LEFT_ARROW = 'ArrowLeft';
+const RIGHT_ARROW = 'ArrowRight';
+const TAB = 'Tab';
```

Comment:
If `case TAB` is removed (see other comment), this line can then also be removed.